rabbithole_test.go

```diff
@@ -1135,6 +1143,48 @@ var _ = Describe("RabbitMQ HTTP API client", func() {
 		})
 	})
 
+	Context("user-limits", func() {
+		maxConnections := 1
+		maxChannels := 2
+		It("returns an empty list of limits", func() {
+			xs, err := rmqc.GetUserLimits("guest")
+			Ω(err).Should(BeNil())
+			Ω(xs).Should(HaveLen(0))
+		})
+		It("sets the limits", func() {
```

Comment:
This creates an example inter-dependency (one cannot pass without another being run first). This is not the case with other examples. If it would makes things easier, several examples can be combined.